### PR TITLE
Update readme

### PR DESCRIPTION
**Pull request by saradickinson**
Willem, please review.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,77 +1,53 @@
...
-
-Logging/debugging
-
-> **`--enable-debug-daemon`** If you don't want to see the connection statistics then remove the `--enable-debug-daemon` option in the `configure` line above.
-
-> **`--enable-debug-stub`**   If you do want to see very detailed debug information as messages are processed (including connection statistics) then add the `--enable-debug-stub` option to the `configure` line above.
+cd stubby
+libtoolize -ci
+autoreconf -vfi
+./configure CFLAGS="-g -I/usr/local/include" LDFLAGS="-L/usr/local/lib" --prefix="$HOME/local"
```
Comment (wtoorop) [inline README.md]: Are you sure about --prefix?

File: README.md [hunk 2]
```diff
@@ -1,77 +1,53 @@
...
-```sh
-> git submodule update --init
-> libtoolize -ci
-> autoreconf -fi
-> mkdir build
-> cd build
-> ../configure --prefix=<install_location> --without-libidn --enable-stub-only --enable-debug-daemon
-> make
-> sudo make install
+Build and install (assuming the getdns is installed in a standard location e.g. by homebrew)
```
Comment (wtoorop) [inline README.md]: What is the standard homebrew location?  /usr/local?

Comment (saradickinson) [inline README.md]: Yes e.g. stubby gets installed in a homebrew cellar and then linked to /usr/local/bin/stubby

File: README.md (hunk 1 above)
Comment (saradickinson) [inline README.md]: Fixed... took the -g out too.

**Pull request merged**